src/components/input/UIInput.vue

```diff
@@ -118,11 +118,19 @@
         </div>
 
         <UIExpandTransition>
-            <slot v-if="error || $slots.error" name="error">
-                <p class="text-color-error text-sm">
-                    {{ error }}
-                </p>
-            </slot>
+            <div v-if="error || $slots.error || help || $slots.help">
+                <slot v-if="error || $slots.error" name="error">
+                    <p class="text-color-error text-sm mt-1">
+                        {{ error }}
+                    </p>
+                </slot>
+
+                <slot v-if="(help || $slots.help) && !error && !$slots.error" name="help">
+                    <p class="text-color-muted text-sm mt-1">
+                        {{ help }}
```

Comment:
Could you add testing please? You'll find other examples how a prop/slot like this is tested